client/tso_service_discovery.go

```diff
@@ -465,7 +465,7 @@ func (c *tsoServiceDiscovery) updateMember() error {
 	oldGroupID := c.GetKeyspaceGroupID()
 	if oldGroupID != keyspaceGroup.Id {
 		log.Info("[tso] the keyspace group changed",
-			zap.Uint32("keyspace-id", keyspaceGroup.Id),
+			zap.Uint32("keyspace-id", keyspaceID),
```

Comment:
Is it necessary to use "keyspace-id-in-request" as in the code above?

Reply:
It's unnecessary because it might printed after finishing the request.